pkg/project/unbind.go

```diff
@@ -20,18 +21,25 @@ import (
 )
 
 // Unbind a project from Codewind
-func Unbind(httpClient utils.HTTPClient, connection *connections.Connection, url, projectID string) error {
+func Unbind(httpClient utils.HTTPClient, connection *connections.Connection, url, projectID string) *ProjectError {
 	req, err := http.NewRequest("POST", url+"/api/v1/projects/"+projectID+"/unbind", nil)
 	if err != nil {
-		return err
+		return &ProjectError{errOpUnbind, err, err.Error()}
 	}
 
-	// send request
 	res, httpSecError := sechttp.DispatchHTTPRequest(httpClient, req, connection)
 	if httpSecError != nil {
-		return httpSecError
+		return &ProjectError{errOpUnbind, httpSecError, httpSecError.Desc}
+	}
+
+	if res.Body != nil {
```

Comment:
You don't need to check that res.Body is nil, it looks like the field is embedded in the response object, not pointed to: https://golang.org/pkg/net/http/#Response
(I only checked this because none of our other `defer res.Body.Close()` do a nil check!)